compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1146,3 +1146,9 @@ error.type.param.outside.lang.module=\
 
 error.versioned.import.not.supported=\
   versioned import is not supported
+
+error.invalid.lvalue.lhs.of.assignment=\
+  invocations are not supported in lhs of an assignment
```

Comment:
how about
"invocations are not supported on the left hand side of an assignment"